content/conditions/earwax/main-content.md

```diff
@@ -0,0 +1,60 @@
+## How you can treat earwax build-up yourself
+
+!!! warning
+  Don’t use your fingers or any objects like cotton buds to remove the earwax. This will push it in further and make it worse.
+!!!
+
+Usually earwax will fall out on its own. If it doesn’t and blocks your ear, put 2 to 3 drops of olive or almond oil in your ear twice a day for a few days. 
+
+Over a week or 2 weeks large lumps of earwax can fall out of your ear, especially at night when you’re lying down.
+
+
+## Your pharmacist can help with earwax 
+
+To dissolve the earwax you can get chemical drops from the pharmacy. 
+
+The earwax should fall out on its own or dissolve after about a week.
+
+Don’t use drops if you have a hole in your eardrum. Ask your pharmacist what you can do instead.
+
+!!! info
+  [Find your nearest pharmacy](https://beta.nhs.uk/finders/find-help?context=stomach-ache)
```

Comment:
Use the global constant for the finders base url: `{{ findersBaseUrl }}`. Also shouldn't contain stomach ache as the context. Maybe check with @st3v3nhunt or @neilbmclaughlin whether they've created a new context for earwax. 

We could still pass earwax as the context now and when implemented on connecting to services it will start picking it up.